rdrf/rdrf/settings.py

```diff
@@ -605,3 +605,6 @@
 
 EXTRA_HIDABLE_DEMOGRAPHICS_FIELDS = ('living_status', )
 LOGIN_LOG_FILTERED_USERS = env.getlist('login_log_filtered_users', ['newrelic'])
+JWT_SECRET_KEY = env.get("jwt_secret_key", "C}>$M1/'#gfe;Y!WSi@%AwRDgaSRe:")
```

Comment:
Please don't set the secret in `settings.py`.
If it is for local development only it goes in an env variable in `docker/envs/runserver`.
Also, don't set any defaults here, which will fail if we forgot to set the env variable.